charts/app/values.yaml

```diff
@@ -34,7 +34,7 @@ formsWebApp:
     name: forms-web-app
     port: 9000
   image:
-    repository: pinscommonukscontainersdefault.azurecr.io/forms-web-app
+    repository: pinscommonukscontainers3887default.azurecr.io/forms-web-app
```

Comment:
curious - why is this numbered now? I see something below:

` name = replace(format(local.name_format, "containers-${random_integer.containers.result}"), "-", "")`

Is it ever possible these get out of sync? 

idk, I may be barking up the wrong bush - curiosity only here

Reply:
Splendid question. The container registry name must be globally unique. As `pinscommonukscontainersdefault` is the name in the F4 subscription, I had to change it to create a separate one in the PINS subscription (I wanted to get it up and running beforehand so I could test it). Even though this will be unique once I drop the F4 one, I've used a random integer in everything that requires a globally unique name.

I don't believe there's any chance of it getting out of sync as the registry is designed to stay up forever - if we drop it then we lose all the container images in the registry.